src/installer.ts

```diff
@@ -197,6 +210,20 @@ async function getInfoFromDist(
   };
 }
 
+async function resolveVersionFromManifest(
+  versionSpec: string,
+  stable: boolean,
+  auth: string | undefined
+): Promise<string | undefined> {
+  try {
+    const info = await getInfoFromManifest(versionSpec, stable, auth);
+    return info?.resolvedVersion;
+  } catch (err) {
+    core.warning('Unable to resolve version from manifest...');
```

Comment:
Should this be a warning?  A warning means the user should take action.  I think we should just output (core.info) so the log is clear what happened but no need for worry and warning.  Seems that's appropriate for later downstream when ultimately a version can't be resolved and that's currently handled.